src/lazy/binary/raw/v1_1/value.rs

```diff
@@ -246,6 +268,37 @@ impl<'top> LazyRawValue<'top, BinaryEncoding_1_1> for &'top LazyRawBinaryValue_1
 }
 
 impl<'top> LazyRawBinaryValue_1_1<'top> {
+    /// Constructs a lazy raw binary value from an input buffer slice that has been found to contain
+    /// a complete `FlexUInt`.
+    pub(crate) fn for_flex_uint(input: ImmutableBuffer<'top>) -> Self {
+        let encoded_value = EncodedValue {
+            encoding: ParameterEncoding::FlexUInt,
+            header: Header {
+                // It is an int, that's true.
+                ion_type: IonType::Int,
+                // Nonsense values for now
+                ion_type_code: OpcodeType::Nop,
+                low_nibble: 0,
```

Comment:
For now? Until when?

Reply:
My intent (not communicated in this comment) was to modify the `EncodedValue` type such that it wouldn't require a `Header` anymore. Matt also [suggested this independently](https://github.com/amazon-ion/ion-rust/pull/801#discussion_r1712904408); I'm going to do it as part of #805.